pages/cart_page.html

```diff
@@ -0,0 +1,177 @@
+</head>
+
+<body>
+    <!-- TOP NAVIGATION -->
+
+    <nav class="top-navbar d-flex">
+        <div class="left">
+            <p class="text-lg">
+                Website Logo
+            </p>
+        </div>
+        <div class="center d-flex flex-justify-center">
+            <input class="search-input" type="text" placeholder="search">
+        </div>
+        <div class="right d-flex flex-justify-evenly flex-align-center">
+            <button class="btn btn-secondary">Log Out</button>
+            <div class="badge">
+                <svg class="badge-icon" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
+                    fill="none" stroke="#000000" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+                    <path
+                        d="M20.84 4.61a5.5 5.5 0 0 0-7.78 0L12 5.67l-1.06-1.06a5.5 5.5 0 0 0-7.78 7.78l1.06 1.06L12 21.23l7.78-7.78 1.06-1.06a5.5 5.5 0 0 0 0-7.78z">
+                    </path>
+                </svg>
+                <span class="badge-md badge-primary">10+</span>
+            </div>
+            <div class="badge">
+                <svg class="badge-icon" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
+                    fill="none" stroke="#000000" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+                    <circle cx="10" cy="20.5" r="1" />
+                    <circle cx="18" cy="20.5" r="1" />
+                    <path d="M2.5 2.5h3l2.7 12.4a2 2 0 0 0 2 1.6h7.7a2 2 0 0 0 2-1.6l1.6-8.4H7.1" />
+                </svg>
+                <span class="badge-md badge-primary">10+</span>
+            </div>
+        </div>
+    </nav>
+
+    <div class="offer-notification text-align-center">Get Free Shipping On Orders Above &#8377; 9999</div>
+
+    <main>
+        <div class="cart-container d-flex">
+            <div class="product-list d-flex">
+                <div class="card card-horizontal d-flex">
+                    <img class="card-horizontal-img"
+                        src="https://res.cloudinary.com/clouduser/image/upload/c_scale,w_198,h_200/v1645367388/download_lym0ig.jpg"
+                        alt="">
+                    <div class="card-data">
+                        <p class="card-title text-xl">The Brown Shoes</p>
+                        <p class="text-sm text-clr-gray">by Some Premium Brand</p>
+                        <p class="text-xl">&#8377; 1999 <span class="text-clr-gray  product-discount">50% OFF</span></p>
+                        <div class="d-flex">
+                            <p>Quantity: </p>
+                            <div class="d-flex">
+                                <p class="text-xl product-qty-control">-</p>
+                                <input class="product-qty-input" type="number">
+                                <p class="text-xl product-qty-control">+</p>
```

Comment:
It will be better if you use button rather than para.

Reply:
done